sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Internals/HttpPipelineHelper.cs

```diff
@@ -207,17 +208,17 @@ internal static ExportResult HandleFailures(HttpMessage httpMessage, PersistentB
 
             if (result == ExportResult.Success)
             {
-                AzureMonitorExporterEventSource.Log.WriteWarning("FailedToTransmit", $"Error code is {statusCode}: Telemetry is stored offline for retry");
+                AzureMonitorExporterEventSource.Log.WriteWarning("FailedToTransmit", $"Error code is {statusCode}: Telemetry is stored offline for retry. Instrumentation Key: {connectionVars.InstrumentationKey}, Configured Endpoint: {connectionVars.IngestionEndpoint}, Actual Endpoint: {httpMessage.Request.Uri.Host}");
```

Comment:
Curious how https://github.com/Azure/azure-sdk-for-net/issues/34483 will work. If we want to include response from breeze

Reply:
I think including the responses from breeze should be out of scope for this change.
I did a network trace for one scenario and errors were returned as an array. 
We'll need to collect a few more examples and think through how to best handle this.